app/src/main/java/com/hamdan/forzenbook/view/login/LoginActivity.kt

```diff
@@ -43,7 +43,7 @@ class LoginActivity : ComponentActivity() {
                                     )
                                 }
                             ) {
-                                loginViewModel.submitLogin()
+                                loginViewModel.requestLoginValidationCode()
```

Comment:
Remember that the View only tells the ViewModel UI related events, like "LoginButtonClicked"

Reply:
Alright I'll address the rest of the viewmodel actions that should occure on button click after this and FA-80 are in, for now I made a button clicked function for this.

Reply:
also introduced a toDO at the top of the viewmodel